### ISSUE TITLE
rename quick links on home page

### ISSUE THREAD

**Issue by tortino** [kiwibrowser/src.next#372]
**Is your feature request related to a problem? Please describe.**

I cannot rename the quick link icons that I have added to the home page.
I added some links to various subreddits, but all icons have the same name:  reddit.com

**Describe the solution you'd like**

I'd like the ability to rename the icons, to set meaningful names.

**Describe alternatives you've considered**

**Additional context**

**Comment by metronightly** [comment]
This would be a really cool addition.

**Comment by MooShuOrc** [comment]
This has been suggested/requested several times already, probably because many users want it. Unfortunately, the devs have made it clear that they're not interested in it, and they suggest using an extension.
I haven't been able to find any NTP extension that isn't bloated/useless and also properly installs/doesn't crash, so if you can find a good one please let the community know.

TL;DR: Yes, that would be great, but it's probably not going to happen.

**Comment by kiwibrowser** [comment]
?
No it's planned and on the todo-list

**Comment by MooShuOrc** [comment]
Really? That great to hear! Sorry if I was mistaken/misinformed.

**Comment by kiwibrowser** [comment]
Oh no problems, it's just that it was not in priority, but of course it makes sense :)

**Comment by d3ward** [comment]
@MooShuOrc 

A good alternative until default ntp get an update can be [Next NTP](https://d3ward.github.io/nextntp/) , highly customizable , not an extension( yet ) but works very well, and could fit your needs

**Comment by MooShuOrc** [comment]
@d3ward

Every time the subject comes up, you suggest bookmarking your website instead. While I appreciate that you're trying to help, it's not really the same. The NTP already takes a strangely long time to load and downloading (and then re-downloading every time) a web page is even worse. Either way, isn't that just a Home Page (which Kiwi supports) instead of a NTP?
I suppose if you have new hardware, a fast connection, and never clear cookies, then your website might be a good replacement. Otherwise (no offense), it's much worse.
Note: sorry if this sounds rude. I know you're trying to help, but it can be frustrating to not have a solution to what should be a simple problem.

**Comment by d3ward** [comment]
> sorry if this sounds rude.

On the contrary you were "fair" and motivated me to finish the migration to extension version of Next NTP i started weeks ago ( a lot faster and looks like native homepage now ! Internet is needed only for news and custom icons )

> it can be frustrating to not have a solution to what should be a simple problem.

Simple problem actually is not , making a native homepage require skills of Android UI and unfortunately is not the best skill of dev
But pretty sure , as dev said , in future ( time is very important for a dev) will be there a native, customizable, fast  and very well done homepage for Kiwi

**Comment by MooShuOrc** [comment]
@kiwibrowser 

I just saw your update to the NTP. Wow! So much better! Thank you thank you thank you!

Seriously, good job. It's very much appreciated.
Keep up the good work.

**Comment by kiwibrowser** [comment]
It's not totally finished yet, but thank you very much for the kind words, I spent lot of time on it (and will continue to do so because I have plenty of ideas on how to make it even better)
